src/archivematicaCommon/lib/archivematica_transfer_types.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+Archivematica transfer types.
+
+Module collects together information related to the validation and initiation
+of Archivematica transfers.
+
+Where feasible this module can be imported to be used in different classes
+calling it in Python. Places where this isn't feasible are as follows:
+
+ - Listing of transfer types plus file-type transfer control:
+     * src/dashboard/frontend/app/browse/browse.controller.js
+
+ - Enable the transfer type to be selected in the browser drop-down:
+     * src/dashboard/frontend/app/front_page/transfer_browser.html
+
+ - Disable the visibility for a transfer name in the browser:
+     * src/dashboard/frontend/app/header/header.controller.js
+
+ - A dummy name is generated for a transfer in the UI:
+     * src/dashboard/frontend/app/services/transfer_browser_transfer.service.js
+
+ - References to watched directories also exist in Workflow.json and MCP
+   server's shared_dirs.py
+     * src/MCPServer/lib/assets/workflow.json
+
+Transfers have different characteristics such as whether a transfer can be
+selected by directory, or by file, e.g. standard transfer vs. zipped bags.
+
+This module should try and capture these characteristics where possible.
+"""
```

Comment:
Adding the only documentation associated with transfer types as they are in Archivematica at present. For looking into the past, and into the future, e.g. to see where legacy wiring needs to be undone.